code/datums/diseases/_MobProcs.dm

```diff
@@ -14,6 +14,9 @@
 	if(D.GetDiseaseID() in resistances)
 		return FALSE
 
+	if(has_trait(TRAIT_NOBREATH) || has_trait(TRAIT_VIRUSIMMUNE))
```

Comment:
we don't want `NOBREATH` species to be immune to contracting any disease; this would means that slime people couldn't suffer from kuru, despite having a brain.

the `VIRUSIMMUNE` check also needs the `!bypasses_immunity` check as well.